client/src/assets/scss/theme.scss

```diff
@@ -27,6 +27,7 @@ $BCgovABlue1: #6e93ff;
 $BCgovABlue2: #3b6cff;
 $BCgovABlue3: #0846ff;
 $BCgovABlue4: #003bee;
+$BCgovABlue5: #1976D2;
```

Comment:
The accent colours are ordered. Could you update please?

Reply:
How to do that? by renaming $BCgovABlue4 -> $BCgovABlue5? I did not understand that

Reply:
Yeah, sorry, I was unclear.

Actually, those "BCGov" colours are from the official BC Gov theme (outside this project). So we should not change it.

I suggest two options:
1. Get Tracey to agree to use one of the existing BC Gov accent colours, or
2. Create a new variable `$app-lt-blue` that we can all reuse.

Reply:
While you're there, could you please use a colour variable for `.bg-light-blue` ? (This is cleanup.)

Reply:
(That way we keep all our colours in one place!)

Reply:
(And it will be easier to find a colour for reuse, and also reduce the number of colours we use!)

Reply:
will do